src/components/SelectInput/SelectInput.module.scss

```diff
@@ -1,15 +1,16 @@
 @import '../../styles/colors';
+@import '../../styles/theme';
 
 .InputContainer {
     min-width: 17rem;
     text-align: left;
 
     .Wrapper:hover {
-        border-color: #8E4EC6;
+        //border-color: $color-;
```

Comment:
nit: is this hover style supposed to be removed?

Reply:
Good point, I don't think so. Let me check!

Reply:
Looks like there's no hover, just `active` and `focus`.